src/mongocrypt-marking.c

```diff
@@ -935,7 +893,7 @@ _mongocrypt_fle2_placeholder_to_insert_update_ciphertextForRange (
       }
    }
 
-   // Serialize.
+   _mongocrypt_ciphertext_init (ciphertext);
```

Comment:
```suggestion
```

Current callers of `_mongocrypt_marking_to_ciphertext` call `_mongocrypt_ciphertext_init` on the passed `ciphertext`. Suggest removing the `_mongocrypt_ciphertext_init` to not attempt to support uninitialized `ciphertext` from being passed.

Suggest adding a comment to `mongocrypt-marking-private.h` to communicate the expectation:
```c
// Callers are expected to initialize `ciphertext` with _mongocrypt_ciphertext_init before calling.
bool
_mongocrypt_marking_to_ciphertext (void *ctx,
                                   _mongocrypt_marking_t *marking,
                                   _mongocrypt_ciphertext_t *ciphertext,
                                   mongocrypt_status_t *status)
   MONGOCRYPT_WARN_UNUSED_RESULT;
```

Reply:
As above, done.